pkgs/development/interpreters/php/default.nix

```diff
@@ -186,14 +186,15 @@ let
               (lib.textClosureList extensionTexts extNames)}
             ${extraConfig}
           '';
-        in
-          symlinkJoin {
+
+          php-with-extensions = symlinkJoin {
             name = "php-with-extensions-${version}";
             inherit (php) version;
             nativeBuildInputs = [ makeWrapper ];
             passthru = {
               inherit buildEnv withExtensions enabledExtensions;
               inherit (php-packages) packages extensions;
+              customConfig = "${php-with-extensions}/lib/custom-php.ini";
```

Comment:
I'm not a big fan of `customConfig` as a name for this and I think the file should probably be renamed too. That said, I think it's a nice addition that I would like to see, but could we do it in or after #85026? It would likely be a lot of work trying to rebase it on top of this, seeing how big the changes are in it..

Reply:
Yeah, I did not like it either but wanted to keep the change minimal. I guess we could hardcode the path to the ini file in the httpd module for now to quickly unbreak this and switch it to `passthru` in #85026.

Reply:
Dropped the commit.

Reply:
Thanks!